### PR TITLE
Misc matches & cleanup

### PR DESCRIPTION
**Pull request by ethteck**

### CODE REVIEW

**Comment by BowserSlug** [comment]
🚀🚀🚀 This PR matches 3 functions (+0.05%) on `us`.

File: src/17D6A0.c [hunk 1]
```diff
@@ -23,47 +23,50 @@ extern ActorBlueprint battle_partner_lakilester;
...
-    3, 11, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0,
-    0, 0, 0, 0, 0, 0,
+UnkPartnerThing D_80280FC0 = {
+    10, 90,
+    {
+        50, 196619,
```
Comment (HailSanta) [inline src/17D6A0.c:29]: The second field here are player anim IDs.
e.g., 196619 = 0x1002D = ANIM_Mario_1002D

Comment (ethteck) [reply src/17D6A0.c:29]: ah nice - I thought that may be the case, but I wasn't sure. I'll add those

File: src/entity_model.c [hunk 2]
```diff
@@ -1,6 +1,7 @@
 #include "common.h"
+#include "entity.h"
 
-s32 D_8014C260[] = { 0x00000000, 0x00000000 };
+EntityModelScript D_8014C260[] = { {0x00000000}, {0x00000000 }};
```
Comment (HailSanta) [inline src/entity_model.c:4]: 0 -> ems_End ?

**Pull request merged**